service/grpc/codelingo/codelingo.pb.go

```diff
@@ -1194,8 +825,39 @@ func (x *codeLingoReviewClient) Recv() (*Issue, error) {
 	return m, nil
 }
 
+func (c *codeLingoClient) Codemod(ctx context.Context, opts ...grpc.CallOption) (CodeLingo_CodemodClient, error) {
+	stream, err := grpc.NewClientStream(ctx, &_CodeLingo_serviceDesc.Streams[1], c.cc, "/codelingo.CodeLingo/Codemod", opts...)
+	if err != nil {
+		return nil, err
+	}
+	x := &codeLingoCodemodClient{stream}
+	return x, nil
+}
+
+type CodeLingo_CodemodClient interface {
```

Comment:
type name will be used as codelingo.CodeLingo_CodemodClient by other packages, and that stutters; consider calling this _CodemodClient